stablehlo/tests/interpret_abs.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: abs_op_test_si32
+func.func @abs_op_test_si32() -> tensor<3xi32> {
```

Comment:
See the [testing guidelines]( https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines). We should use the highest precision of the given type. Here and below.